### PR TITLE
[AIRFLOW-XXXX] Review and cleanup integration tests docs.

### PR DESCRIPTION
**Pull request by potiuk**
---
Issue link: `Document only change, no JIRA issue`

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
Hey @efedotova - here is the PR that you can use to review/improve the integration tests docs :)

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -289,8 +289,8 @@ We support the following types of tests:
 
 * **Integration tests** are available in the Breeze development environment
   that is also used for Airflow Travis CI tests. Integration test are special tests that require
-  additional services running, such as Postgres,Mysql, Kerberos, etc. These tests are not yet
-  clearly marked as integration tests but soon they will be clearly separated by the ``pytest`` annotations.
+  additional services running, such as Postgres,Mysql, Kerberos, etc. They are marked using
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
  additional services running, such as Postgres, Mysql, Kerberos, etc. They are marked using
```

**Review by turbaszek — CHANGES_REQUESTED** [review]

**Comment by potiuk** [comment]
@efedotova ? Any comments :) ?

**Comment by potiuk** [comment]
The documentation changes have been already implemented in #7388

**Pull request closed without merging**